english/formosa/taiwan-story.md

```diff
@@ -0,0 +1 @@
+Did you know that Taiwan has its own "Anne Frank" story about a man in Hsinchu who hid in a small, secret hiding place -- a thin space between two walls, with no room to even stand up -- for 18 years during the White Terror period?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.